public/teams_selectpage.php

```diff
@@ -0,0 +1,56 @@
+<?php
+require_once "../modules/models/user.php";
+require_once "../common.php";
+require_once "../modules/models/team.php";
+require_once "../modules/models/team_member.php";
+require_once "../modules/models/lecture.php";
+require_once "../modules/models/section_student.php";
+require_once "../modules/models/section.php";
+?>
+
+<?php include "templates/header.php"; ?>
+
+<h2>My Teams</h2>
+
+<?php
+
+// Get the current user
+$current_user = $_SESSION["current_user"];
+
+// Get current user role
+$role = get_current_role();
+
+// TODO: Once tas are added, handle team selection here
+if ($role == "instructor" && !$current_user->is_admin) {
+    $all_info = Lecture::joins_raw_sql("
+        JOIN lecture_instructors li on
+        li.lecture_id = lectures.id
```

Comment:
maybe we should go with `all_lectures` or even just `lectures`

Reply:
yeah instead of that, it just feels like an unintuitive name, if you look at the variable name alone, you'd probably not be able to guess what it contains